fe/fe-core/src/main/java/org/apache/doris/resource/resourcegroup/ResourceGroup.java

```diff
@@ -69,6 +77,11 @@ public class ResourceGroup implements Writable {
 
     private double memoryLimitPercent;
 
+    private QueryQueue queryQueue;
+    private int maxConcurrency = Integer.MAX_VALUE;
+    private int maxQueueSize = 0;
```

Comment:
If `maxQueueSize` is set to 10, and there are 3 FE. So the max queue is 10 or 30?

Reply:
30